web/models/user_from_auth.ex

```diff
@@ -183,7 +187,7 @@ defmodule Parma.UserFromAuth do
       auth.info.name
     else
       [auth.info.first_name, auth.info.last_name]
-      |> Enum.filter(&(&1 != nil and String.strip(&1) != ""))
+      |> Enum.filter(&(&1 != nil and String.trim(&1) != ""))
```

Comment:
is there a reason why you prefer `trim` over `strip`

Reply:
`strip` is deprecated
https://github.com/elixir-lang/elixir/blob/v1.3/CHANGELOG.md#3-soft-deprecations-no-warnings-emitted